service-mango-orderbook/src/main.rs

```diff
@@ -60,6 +62,25 @@ pub enum Command {
 #[serde(rename_all = "camelCase")]
 pub struct SubscribeCommand {
     pub market_id: String,
+    pub subscription_type: SubscriptionType,
```

Comment:
Should be an Option and default to SubscriptionType::Level if omitted to remain compatible with existing clients